File: core/src/main/java/org/c0nstexpr/fishology/core/config/FishologyConfigModel.java
package org.c0nstexpr.fishology.core.config;

import co.touchlab.kermit.Severity;
import io.wispforest.owo.config.annotation.Config;
import io.wispforest.owo.config.annotation.Modmenu;
import org.c0nstexpr.fishology.core.FishologyCoreModKt;

@Modmenu(modId = FishologyCoreModKt.modId)
@Config(name = FishologyCoreModKt.modId, wrapperName = "FishologyConfig")
public class FishologyConfigModel {
    public boolean enabled = true;

    public Severity logLevel = Severity.Warn;
}

File: src/main/kotlin/org/c0nstexpr/fishology/FishologyConfigModel.kt
package org.c0nstexpr.fishology

import io.wispforest.owo.config.annotation.Config
import io.wispforest.owo.config.annotation.Modmenu

@Modmenu(modId = "fishology")
@Config(name = "fishology-config", wrapperName = "FishologyConfig")
class FishologyConfigModel {
    var enabled = true
}
